src/main/kotlin/apply/ui/api/ApplicationFormRestController.kt

```diff
@@ -19,36 +19,23 @@ class ApplicationFormRestController(
 ) {
     @GetMapping
     fun getForm(@RequestParam("recruitmentId") recruitment: Long): ResponseEntity<Any> {
```

Comment:
```suggestion
    fun getForm(@RequestParam("recruitmentId") recruitment: Long): ResponseEntity<ApplicationFormResponse> {
```
이젠 예외 상황은 따로 핸들링해서 Any를 없애줄 수 있네요